woocommerce/woocommerce
# Add initial release of WooAI plugin

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Initial release of Woo AI plugin, which includes the product name suggestions and description generation features.



### How to test the changes in this Pull Request:

1. The current state of this plugin only works within Atomic. I've been using a  WoA dev site ([guide](pb5gDS-1rQ-p2)). Following that guide, but with [this modified script](https://gist.github.com/joelclimbsthings/d0144f525e165d4943650feeec09c9a4).
2. Once you've copied the plugin over, ensure the `Woo AI` plugin is activated.
3. Go to Products -> Add new
4. Click on the product title field to explore the product name suggestions feature.
5. Click the "Write with AI" button above the product description to explore the description generation feature.
